spring-boot/proxy/src/main/resources/application.yml

```diff
@@ -35,3 +35,13 @@ logging:
   level:
     eu.coatrack.proxy: DEBUG
     org.springframework.security: DEBUG
+
+# The gateway has the ability to work even if the CoatRack admin server is not reachable. If admin is not reachable
+# for longer than this period of time, the gateway will reject all service API requests until a connection could
+# be re-established.
+number-of-minutes-the-gateway-works-without-connection-to-admin: 60
+
+# The gateway periodically updates its local API key list which is used for authorization of incoming service
+# requests if the connection to the CoatRack admin server does not work. This is the time interval for these
+# updates. The default value is 300,000 = 5 minutes.
```

Comment:
This is misleading, it sounds like "if you put no value here it will be 5 minutes". But it looks like it is mandatory to put a value here. Thus I would suggest to remove this.

Reply:
Done.